### ISSUE TITLE
Save displays as '...' and stays that way indefinitely

### ISSUE THREAD

**Issue by jsonpj3wt** [strapi/strapi#4550]
**Describe the bug**
This is very inconsistent to reproduce but the most likely scenario is to go to a model that has relationships.  Go to the relationship and attempt to edit it.  Eventually, the `SAVE` button will never show and is the `...` loading indicator.

**Steps to reproduce the behavior**
1. Go to a model with relationships.
2. Click on the relationship to edit.
3. See error

**Expected behavior**
The `Save` button show show up and honestly, probably should not make content editable until that point.  Otherwise, users will enter data and all of a sudden lose every change because the `Save` button is not enabled.

**Screenshots**
![image](https://user-images.githubusercontent.com/44671008/69098579-7cf5c680-0a1e-11ea-86a3-f5e652c93a6a.png)

**System**
- Node.js version: 10.16.0
- NPM version: 6.9.0
- Strapi version: v3.0.0-beta.17.5
- Database: MongoDB
- Operating system: Linux AMI

**Other**
Only thing I can see from console/network when getting this error is this:
`Refused to load the image 'data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==' because it violates the following Content Security Policy directive: "img-src 'self' http:".`

**Comment by lauriejim** [comment]
Can you send your data model please.

**Comment by jsonpj3wt** [comment]
This happens on a few components but here is the one I got the error with in production:
**page.settings.json**
`{
  "connection": "default",
  "collectionName": "page",
  "info": {
    "name": "page",
    "description": ""
  },
  "options": {
    "timestamps": [
      "createdAt",
      "updatedAt"
    ]
  },
  "attributes": {
    "title": {
      "required": true,
      "type": "string"
    },
    "tags": {
      "collection": "tag",
      "via": "pages",
      "dominant": true
    },
    "url": {
      "unique": true,
      "required": true,
      "type": "string"
    },
    "favorites": {
      "collection": "favorite",
      "via": "page"
    },
    "previewImage": {
      "model": "file",
      "via": "related",
      "plugin": "upload"
    },
    "mainmenusubmenuitems": {
      "collection": "mainmenusubmenuitem",
      "via": "page"
    },
    "mainmenuitems": {
      "collection": "mainmenuitem",
      "via": "page"
    },
    "categories": {
      "collection": "category",
      "via": "pages",
      "dominant": true
    },
    "featuredUntil": {
      "type": "date"
    },
    "previewText": {
      "type": "text"
    },
    "likes": {
      "collection": "like",
      "via": "page"
    },
    "isPost": {
      "type": "boolean",
      "required": true,
      "default": true
    },
    "comments": {
      "collection": "comment",
      "via": "page"
    },
    "allowComments": {
      "default": true,
      "type": "boolean"
    },
    "featuredStart": {
      "type": "date"
    },
    "pagesidenav": {
      "model": "pagesidenav",
      "via": "pages"
    },
    "accordions": {
      "collection": "accordion",
      "via": "pages"
    },
    "imagegallerywidgets": {
      "collection": "imagegallerywidget",
      "via": "page"
    },
    "imagewidgets": {
      "collection": "imagewidget",
      "via": "page"
    },
    "legacyhtmlwidgets": {
      "collection": "legacyhtmlwidget",
      "via": "page"
    },
    "richtextwidgets": {
      "collection": "richtextwidget",
      "via": "page"
    },
    "videos": {
      "collection": "video",
      "via": "page"
    }
  }
}`

There are a ton of relationships to this (being a page and all) so if you need those, can you PM me in Slack and I can send you all of the models?

**Comment by jsonpj3wt** [comment]
So I have a hunch of what is going on here... We are using AWS ECS/Fargate clusters and what I *think* is happening is a request is made and it is getting load-balanced.  I will occasionally get 503 Gateway errors which makes me think that might be the problem.  What I don't get is why that is an issue?  The token should be valid so not sure why this happening?

**Comment by jsonpj3wt** [comment]
I am also able to reproduce this on localhost as well.  It looks like a bug in the UI... I am able to configure the button to *NOT* be disabled inspecting it and removing _disabled_ and everything works fine.

**Comment by lauriejim** [comment]
Here is a video about the issue -> https://www.loom.com/share/861655cc0750493aa98df8f71ba53cd9

Will be easier to understand the issue.

**Comment by hynding** [comment]
I was able to fix the issue for v17.5 as I had already made some custom view updates to the `EditView` container. 

Looking at the current master branch (3.0.0-beta.18.7) the `handleSubmit` method has been moved to the EditViewDataManagerProvider container. The first nested `try`/`catch` in `handleSubmit` has `setIsSubmitting(false);` in the `catch` but not the `try`. If you move the `setIsSubmitting(false);` out of and after that catch (just before the "parent" `catch`, in the "parent" `try`) it should work.

I'm more than happy to commit and submit a PR; before I do, does this fix warrant a new test?
